### PR TITLE
Update Android NDK version and API version

### PR DESCRIPTION
**Pull request by dalance**
This PR updates Android NDK version and API version as the same as libc crate.

https://github.com/rust-lang/libc/blob/master/ci/android-install-ndk.sh

Fix https://github.com/rust-embedded/cross/issues/195

### CODE REVIEW

**Review by burrbull — APPROVED** [review]

**Comment by dalance** [comment]
I've updated android-system.sh to update bionic versiion to Andriod 9.0.0.
Docker build was success at my local environment.

**Comment by dalance** [comment]
bors try

**Comment by dalance** [comment]
@burrbull  Could you retry test?

**Comment by svenstaro** [comment]
bors try

**Comment by Emilgardis** [comment]
bors try

**Comment by dalance** [comment]
Could you retry test?

File: ci/test.sh [hunk 1]
```diff
@@ -148,7 +148,7 @@ EOF
     if (( ${CPP:-0} )); then
         td="$(mktemp -d)"
 
-        git clone --depth 1 https://github.com/japaric/hellopp "${td}"
+        git clone --depth 1 https://github.com/dalance/hellopp -b update_cc "${td}"
```
Comment (reitermarkus) [inline ci/test.sh:151]: Should we fork this in the `cross-rs` org instead?

**Comment by reitermarkus** [comment]
bors try

File: ci/test.sh (hunk 1 above)
Comment (dalance) [reply ci/test.sh:151]: I think so.

Comment (reitermarkus) [inline ci/test.sh:151]:
```suggestion
        git clone --depth 1 https://github.com/cross-rs/rust-cpp-hello-word "${td}"
```

File: ci/test.sh [hunk 2]
```diff
@@ -148,7 +148,7 @@ EOF
...
 
         pushd "${td}"
         cargo update -p gcc
```
Comment (reitermarkus) [inline ci/test.sh:154]:
```suggestion
```

**Review by reitermarkus — DISMISSED** [review]

**Comment by reitermarkus** [comment]
bors try --target android

**Comment by reitermarkus** [comment]
bors cancel

**Comment by reitermarkus** [comment]
bors try --target android

**Comment by dalance** [comment]
I think this PR is ready to merge. Could you review?

**Comment by Emilgardis** [comment]
Please rebase the branch instead of using merge commit 👍

If you are unsure how, just ask!

**Comment by dalance** [comment]
I removed merge commits.
How about it?

File: src/docker.rs [hunk 3]
```diff
@@ -168,6 +168,7 @@ pub fn run(
     docker.args(&["-e", "PKG_CONFIG_ALLOW_CROSS=1"]);
 
     docker.arg("--rm");
+    docker.arg("--privileged");
```
Comment (Emilgardis) [inline src/docker.rs:171]: is this really needed for all targets? Seems wrong

**Review by Emilgardis — CHANGES_REQUESTED** [review]

File: src/docker.rs (hunk 3 above)
Comment (dalance) [reply src/docker.rs]: This is needed for 32bit android only.
I added `needs_docker_privileged` to judge it.

**Review by Emilgardis — APPROVED** [review]
thanks!

bors r+

**Comment by dalance** [comment]
@reitermarkus  Could you check the change requests?

**Review by Emilgardis — APPROVED** [review]
bors r+

**Comment by dalance** [comment]
I resolved conflict.

**Review by Emilgardis — APPROVED** [review]
bors r+

**Pull request merged**